### ISSUE TITLE
Failed to build PDF using 1.5a2

### ISSUE THREAD

**Issue by shkumagai** [sphinx-doc/sphinx#3062]
I failed to build PDF using 1.5a2 though I success to build from same source using 1.4.8.

added settings:
- language = 'ja'
- latex_docclass = {'manual': 'jsbook'}

When I tried it, got these message:

```
! Undefined control sequence.
l.895   \hypersetup
                   {setpagesize=false}
?
```

It said 'undefined control sequence'.

**Comment by jfbu** [comment]
Thank you for reporting and apologies for this!

The cause is at https://github.com/sphinx-doc/sphinx/blob/master/sphinx/texinputs/sphinx.sty#L895, because `hyperref` loading has been moved to later at PR #3030, but this line was overlooked. Sorry about that.

To temporarily fix this issue in your projects you can use in the  `conf.py`:

``` python
latex_elements = {
     # The paper size ('letterpaper' or 'a4paper').
     #
     # 'papersize': 'letterpaper',
'passoptionstopackages': r'''
\def\hypersetup#1{\let\hypersetup\relax}
\PassOptionsToPackage{setpagesize=false}{hyperref}''',

     # The font size ('10pt', '11pt' or '12pt').
     #
     # 'pointsize': '10pt',

     # Additional stuff for the LaTeX preamble.
     #
     # 'preamble': '',

     # Latex figure (float) alignment
     #
     # 'figure_align': 'htbp',
}
```

A fix will be pushed later today to master branch.

**Comment by shkumagai** [comment]
Thank you for quick reply.
Above way worked fine in my project, so I got a PDF file ;-)
Thanks!

**Comment by jfbu** [comment]
@shkumagai Thanks again for reporting, the issue was closed automatically due to commit  039c68d fixing it. Please re-open if you have further issues.
